release-content/migration-guides/archetype_query_data.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "`ArchetypeQueryData` trait"
+pull_requests: []
+---
+
+To support richer querying across relations,
+Bevy now supports query data that may filter out individual entities.
```

Comment:
```suggestion
Bevy now supports query data that are not archetypal: the query can return entities based on conditions that do not exclusively involve the entity's archetype.

An example of non-archetypal filter is `Changed<C>`: the entity is filtered based on the archetype (having the component C) but also based on the change ticks of the component.
```

I frequently get confused on what 'archetypal' means so maybe this could be useful

Reply:
> I frequently get confused on what 'archetypal' means so maybe this could be useful

I think part of the confusion is that it gets used for wildly different things that happen to be about archetypes, like `Access::archetypal` :).